modules/order/src/Entity/Order.php

```diff
@@ -51,6 +51,7 @@
  *     "canonical" = "/admin/commerce/orders/{commerce_order}",
  *     "edit-form" = "/admin/commerce/orders/{commerce_order}/edit",
  *     "delete-form" = "/admin/commerce/orders/{commerce_order}/delete",
+ *     "reassign-form" = "/admin/commerce/orders/{commerce_order}/reassign",
```

Comment:
I used it for

````
$reassign_route_info = $order->toUrl('reassign-form');
````

Also wasn't sure if we'd add this to operations dropbutton, which simplifies referencing the route then